src/main/java/uk/gov/companieshouse/api/strikeoffobjections/controller/ObjectionController.java

```diff
@@ -49,6 +49,7 @@
     private static final String ERIC_AUTHORISED_USER = "ERIC-Authorised-User";
     private static final String OBJECTION_NOT_FOUND = "Objection not found";
     private static final String ATTACHMENT_NOT_FOUND = "Attachment not found";
+    public static final String COULD_NOT_DELETE = "Could not delete attachment";
```

Comment:
Suggest getting 'ATTACHMENT' into this constant name for some context, like the others above. ATTACHMENT_NOT_DELETED perhaps?